Prox/Prox/Map View/MapViewController.swift

```diff
@@ -3,14 +3,65 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import UIKit
+import GoogleMaps
 
 class MapViewController: UIViewController {
 
-    init() {
-        super.init(nibName: nil, bundle: nil)
-    }
+    lazy var rootContainer: UIStackView = {
```

Comment:
Also, you can make these non-`lazy` (since they'll be created immediately anyway), which will allow you to make them immutable.